README.md

```diff
@@ -31,9 +31,14 @@ The target version of OpenMRS Core.
 ## Release Notes
 
 ### Version 1.7.0
-* Fixed  lab test datatypes, from Free Text to Numeric.
+#### Updates
 * Added Odoo Sale Delivery Status add-on.
 * Added Odoo Sale Payment Status add-on.
+#### Fixes
+* Fixed  lab test datatypes, from Free Text to Numeric.
+* Synced DOB and Patient ID of patients created prior 1.6.0 to Odoo
+* Correctly Mapped Sickle Cell and Cardiology Visit Patients to Respective Patient Lists in French locale
```

Comment:
Remove title case:

> Correctly mapped sickle cell and cardiology visit patients to respective patient lists in French locale

Reply:
Hi @rbuisson I lack the ability make some edits to this repo

Reply:
@rbuisson requested fix done.